### ISSUE TITLE
Shaple of *.npy file?

### ISSUE THREAD

**Issue by egg347** [deepmind/kinetics-i3d#21]
In the sample code,the example video has been preprocessed, with RGB and Flow NumPy arrays provided.
I want to test my own video, so I consider it might be a way to generate my own Numpy arrays and replace the example ones.
For RGB, The provided *.npy file has shape (1, num_frames, 224, 224, 3). It seems that 'num_frames' means number of frames, '224,224' means heights and widths, ‘3’ means channel(RGB). I'm coufused about ‘1’, what does this mean?And its value?

By the way, what's the equation of norm of the logits tensor?

**Comment by poomdla** [comment]
1 means batch_size
